src/DssSpell.sol

```diff
@@ -69,6 +71,9 @@ contract DssSpellAction is DssAction {
 
     function actions() public override {
 
+        // Uncomment to include changes from the DssSpellCollateralOnboardingAction
+        // onboardNewCollaterals();
```

Comment:
Can this be called in the constructor for the Collateral contract? 
Then it will automatically get executed when it's instantiated and we won't need to uncomment here. Should still be a no-op if there's nothing in the function.

Reply:
No, I guess it can't because it gets executed later.

Reply:
A modifier on `actions()` might do the trick though.

```
modifier collateralActions() {
    onboardNewCollaterals(); 
    _;
}

function actions() public collateralActions override {
. . . 
```

something like that? Then we won't even have to mess with it.

Reply:
just leaving it uncommented in the actions function does not seem to cost much gas, but I didn't like calling "onboard" each time even when we weren't onboarding

```haskell
Running 13 tests for src/DssSpell.t.sol:DssSpellTest
[PASS] testFail_notScheduled() (gas: 7161)
[PASS] test_auth_in_sources() (gas: 281473904838271)
[PASS] test_use_eta() (gas: 326798)
[PASS] testOnTime() (gas: 2688526)
[PASS] testVestTransfer() (gas: 4063792)
[PASS] test_nextCastTime() (gas: 327693)
[PASS] testFailTooEarly() (gas: 3528)
[PASS] test_bytecode_matches() (gas: 2785336)
[PASS] test_auth() (gas: 281473913169994)
[PASS] testSpellIsCast_GENERAL() (gas: 11539023)
[PASS] testFailWrongDay() (gas: 3572)
[PASS] testCastCost() (gas: 2690137)
[PASS] testFailTooLate() (gas: 3549)
```

semantically, i like the idea of having a diff that shows when we are onboarding Collateral in the main spell beyond just the changes to the onboarding actions contract since that might happen in a different PR or at a different time